lib/nuget-parser/csproj-parser.ts

```diff
@@ -24,23 +24,27 @@ export async function getTargetFrameworksFromProjFile(
 
     const csprojContents = fs.readFileSync(csprojPath);
 
-    let frameworks: (TargetFramework | undefined)[] = [];
+    let targetFrameworks: (TargetFramework | undefined)[] = [];
     parseXML.parseString(csprojContents, (err, parsedCsprojContents) => {
       if (err) {
         reject(new FileNotProcessableError(err));
         return;
       }
-      const versionLoc = parsedCsprojContents?.Project?.PropertyGroup?.[0];
-      const versions = []
-        .concat(
-          (
-            versionLoc?.TargetFrameworkVersion?.[0] ||
-            versionLoc?.TargetFramework?.[0] ||
-            versionLoc?.TargetFrameworks?.[0] ||
-            ''
-          ).split(';'),
-        )
-        .filter(Boolean);
+      const versions = parsedCsprojContents?.Project?.PropertyGroup?.reduce(
```

Comment:
nitpick: flat map here may be more readable than the reduce,

Reply:
like it @JamesPatrickGill , updated 👍